% update chapters
% update NSF grant
%________________________________________________________________________________________
% @brief    LaTeX2e Resume for C. Titus Brown
% @author   Titus Brown
% @date     January 2008
% @info     Based on Latex Resume Template by Chris Paciorek 
%           http://www.biostat.harvard.edu/~paciorek/


%________________________________________________________________________________________
\documentclass[margin,line]{resume}

\usepackage{hyperref}
\usepackage{color}
\hypersetup{urlcolor=blue}

\begin{document}
\name{\Large C. Titus Brown}
\pagenumbering{arabic}
\pagestyle{plain}


\begin{resume}


    %____________________________________________________________________________________

    {\small (As of June 2020.)}

    % Education
    \section{\mysidestyle Education}

    {\bf Reed College}, Portland, OR; Mathematics; B.A., 1997

    \vspace{2mm}

    {\bf California Institute of Technology}, Davidson Lab (graduate student);
\\
Developmental Biology; PhD., 2007

    \vspace{2mm}

    {\bf California Institute of Technology}, Bronner-Fraser Lab (postdoc);\\
 Developmental Biology and Bioinformatics; 2007-2008

    \vspace{2mm}

    \section{\mysidestyle Appointments}

    {\bf Assistant Professor}, 
Microbiology \& Molecular Genetics / Computer Science and Engineering\\
Michigan State University, 2008-2014.

    {\bf Associate Professor}, 
Population Health and Reproduction, School of Veterinary Medicine \\
University of California Davis, 2015-present.

    %____________________________________________________________________________________
    % Honours and Awards
    \section{\mysidestyle Honours and\\Awards} 

Burroughs-Wellcome Fund Computational Biology Fellowship (1999-2004).\\
Withrow Award for Teaching Excellence in Computer Science at MSU (2008-2009). \\
Woods Hole Marine Biological Laboratory Summer Fellow (2013).\\
Michigan State University / College of Natural Science Teacher-Scholar Award (2013).\\
{\bf Moore Foundation Data Driven Discovery Investigator (2014-2019).}


\section{\mysidestyle Awards}

20. NSF DBI-2018911 \$297,922 (100\%), 3 years, 8/20-7/23, notified of funding but pending award receipt. ``BBSRC-NSF/BIO:Collaborative Research: genomeRxiv: a microbial whole-genome database and diagnostic marker design resource for classification, identification, and data sharing.'' co-PI on collaborative proposal with Vinatzer and Heath (VT); amount is sum to UC Davis.

19. USDA 2020-67015-31675 \$498,356 (100\%) 3 years, 3/20-2/23. ``Tools and resources for cattle pangenomics.'' PI.

18. NIH 1OT3OD025459, \$1,237,582 (100\%); 12 months, 1/20-12/20. ``The Common Fund Data Ecosystem'', 2nd year funding. Subaward via U. Maryland, PI Owen White. I am considered full co-PI by NIH; amount is sum to UC Davis.

17. NIH 1OT3OD025459, \$374,645 (100\%); 8 months, 4/19-12/19. ``The Common Fund Data Ecosystem'', 1st year funding. Subaward via U. Maryland, PI Owen White. I am considered full co-PI by NIH; amount is sum to UC Davis.

16. NIH OT3OD025465, \$1.9m (100\%); 18 months, 10/17-3/19. NIH Data Commons Consortium Pilot Phase. Sole PI on 1 of 10 awards for establishing a pilot consortium; this award was for coordination, training, and engagement.

15. Department of Homeland Security, \$248,000 (100\%); 6/17-9/18. Subaward (\$248k is amount to UC Davis) ``High Confidence Metagenomics Analysis of Complex Samples.''

14. Sloan Foundation, \$20,000 (100\%); 7/1/16-12/31/16. ``A Workshop on Dockerized Notebook Computing.'' Sole PI.

13. Moore Foundation GBMF4551, \$1,750,000 (100\%); 12/1/14-12/1/21.  ``Software to support data driven inquiry in biology.'' Investigator award, sole PI.

12. USDA, \$129,889 (100\%); 10/1/13-9/30/16.  Collaborative proposal with WUSTL. ``Improvement of the Chicken Reference Genome.''

{\centerline {Page 1/9}}

\newpage

11. NSF \#1319909, \$273,419 (20\%); 9/2013-9/2016. co-PI, ``Supporting Efficient Discrete Box Queries for Sequence Analysis on Large Scale Genome Databases.''

10. NIH R25HG006243, \$162,000 (100\%); 7/2013-7/2016.  Sole PI, ``Analyzing Next-Generation Sequencing Data.'' Renewal.

9. NIH R01HG007513, \$699,231 (100\%). 5/2013-5/2016. Sole PI, ``BIGDATA: Low-Memory Streaming Prefilters for Biological Sequencing Data.''

8. USDA, \$2,989,032 (5.8\%). 1/2013-1/2017.  co-PI, ``The Genetics of Johne's Disease.''

7. NSF OCI Supplement to BEACON STC, \$200k (100\%); 1/1/2013-12/31/2013.  Sole PI, ``Materials and Workshops for Cyberinfrastructure Education in Biology.''

6. USDA (Grad Fellowships), \$238,000 (15\%).  1/2012-11/2017.  co-PI, ``Integrated genomics training program.''

5. DOE, \$659,587 (5\%).  9/2011-8/2012.  co-PI, ``Ribosomal Database Project.''

4. NIH R25HG006243, \$104,000 (100\%); 7/2011-6/2014.  Sole PI, ``Analyzing Next-Generation Sequencing Data.''

3. USDA, \$99,000; 2/2009-12/2011 (100\%).  PI, ``Positional Candidate Genes for Resistance to Marek's
Disease by Screening for Marek's Disease Virus Meq-regulated Genes.''

2. NSF, \$50,000; 9/2009-8/2011 (100\%).  PI, ``RV1: MSB: Collaborative: Symbiont Separation and Investigation of the Novel Heterotrophic Osedax Symbiosis using Comparative Genomics.''

1. USDA, \$690,000; 12/2009-11/2013 (100\%).  PI, ``Easily accessible Web-based tools for analyzing next-generation sequencing data from agricultural animals.''

    %____________________________________________________________________________________
% Publications

    \section{\mysidestyle Preprints}

Note: {\color{red} ${\bf \triangleright}$} highlights the most important
papers, in my opinion (personal contribution/impact).

{\em Whole genome assembly of Culex tarsalis.} BJ Main, {\bf CT Brown}, M Marcantonio, CM Barker. bioRxiv.

{\em Draft genome assemblies using sequencing reads from Oxford Nanopore Technology and Illumina platforms for four species of North American killifish from the Fundulus genus}, LK Johnson, R Sahasrabudhe, T Gill, J Roach, L Froenicke, {\bf CT Brown}, A Whitehead. BioRxiv.

{\em Microbial contamination in the genome of the domesticated olive.} T Reiter, {\bf CT Brown}. bioRxiv.

{\color{red} ${\bf \triangleright}$}
{\em A Reference-Free Algorithm for Computational Normalization of Shotgun Sequencing Data.} {\bf Brown CT}, Howe AC, Zhang Q, Pyrkosz AB, Brom TH. 2012. arXiv:1203.4802.

{\em Crossing the streams: a framework for streaming analysis of short DNA sequencing reads}. QP Zhang, S Awad, {\bf CT Brown}. 2015. https://peerj.com/preprints/890/.

{\em Evaluating Metagenome Assembly on a Simple Defined Community with Many Strain Variants}. S Awad, LC Irber, CT Brown. 2017. https://www.biorxiv.org/content/early/2017/07/03/155358.

\vspace{1cm}
{\centerline {Page 2/9}}

\newpage

\section{\mysidestyle Peer Reviewed Publications}

{\em Full publication list at: http://scholar.google.com/citations?user=O4rYanMAAAAJ}

{\color{red} ${\bf \triangleright}$}
{\em Exploring neighborhoods in large metagenome assembly graphs reveals hidden sequence diversity.} {\bf CT Brown}, D Moritz, M O'Brien, F Reidl, T Reiter, B Sullivan. ACCEPTED, Genome Biology, May 2020. (Available on biorxiv.)

{\em A phylogenetically novel cyanobacterium most closely related to Gloeobacter}. CL Grettenberger et al.
The ISME Journal, 1-11

{\em Whole genome sequencing for mutation discovery in a single case of lysosomal storage disease (MPS type 1) in the dog}. TA Mansour et al. 
Scientific Reports 10 (1), 1-10

{\em Genomic and phenotypic analyses of six offspring of a genome-edited hornless bull}. AE Young et al. Nature biotechnology 38 (2), 225-232

{\em FAIRshake: Toolkit to Evaluate the FAIRness of Research Digital Resources}. DJB Clarke et al.
Cell systems 9 (5), 417-421

{\em Sequencing data discovery with MetaSeek}. A Hoarfrost, N Brown, {\bf CT Brown}, C Arnosti. Bioinformatics 35 (22), 4857-4859

{\em Kevlar: a mapping-free framework for accurate discovery of de novo variants.} DS Standage, {\bf CT Brown}, F Hormozdiari. iScience 18, 28-36

{\em Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2.} E Bolyen et al.
Nature biotechnology 37 (8), 852-857

{\color{red} ${\bf \triangleright}$}
{\em Large-scale sequence comparisons with sourmash.} NT Pierce, L Irber, T Reiter, P Brooks, {\bf CT Brown}. F1000Research 8, 1006.

{\em Re-assembly, quality evaluation, and annotation of 678 microbial eukaryotic reference transcriptomes.} LK Johnson, H Alexander, {\bf CT Brown}. GigaScience 8 (4), giy158.

{\em Review, evaluation and directions for gene-targeted assembly for ecologic analyses of metagenomes}. J Guo, J Quensen, Y Sun, Q Wang, {\bf CT Brown}, JR Cole, JM Tiedje. Frontiers in Genetics 10, 957.

{\em VA-Store: A Virtual Approximate Store Approach to Supporting
  Repetitive Big Data in Genome Sequence Analyses.} X Liu, Q Zhu, S
Pramanik, {\bf CT Brown}, G Qian.  IEEE Transactions on Knowledge and Data
Engineering

{\em Whole genome variant association across 100 dogs identifies a frame shift mutation in DISHEVELLED 2 which contributes to Robinow-like syndrome in Bulldogs and related screw tail dog breeds.} TA Mansour et al.
PLoS genetics 14 (12), e1007850.

{\em A Missense Mutation in the Vacuolar Protein Sorting 11 (VPS11) Gene Is Associated with Neuroaxonal Dystrophy in Rottweiler Dogs.} KL Lucot et al.
G3: Genes, Genomes, Genetics 8 (8), 2773-2780.

{\em Sustainable computational science: the ReScience initiative}. NP Rougier et al, PeerJ Computer Science 3, e142.

{\em Draft genome of tule elk Cervus canadensis nannodes}. JE Mizzi, ZT Lounsberry, CT Brown, BN Sacks. F1000Research 6.

\vspace{2cm}
{\centerline {Page 3/9}}

\newpage

{\em Identification of long non-coding RNA in the horse transcriptome} EY Scott et al., CJ Finno. BMC genomics 18 (1), 511.

{\em Central nervous system transcriptome of Biomphalaria alexandrina, an intermediate host for schistosomiasis} TA Mansour et al., MW Miller. BMC research notes 10 (1), 729.

{\em FGF4 retrogene on CFA12 is responsible for chondrodystrophy and intervertebral disc disease in dogs.} EA Brown et al., DL Bannasch. PNAS 2017/10/11.

{\em Evolutionary loss of melanogenesis in the tunicate Molgula occulta.} C Racioppi et al., F. Ristoratore. EvoDevo 2017.

{\em sourmash: a library for MinHash sketching of DNA}. CT Brown and LC Irber. Journal of Open Source Software, 1(5), 27, doi:10.21105/joss.00027.

{\em Centromere location in Arabidopsis is unaltered by extreme divergence in CENH3 protein sequence.} S Maheshwari, T Ishii, CT Brown, A Houben, L Comai. Genome Research 27 (3), 471-478. 2017.

{\em khmer release v2.1: software for biological sequence analysis.} Standage et al., CT Brown. Journal of Open Source Software, 2(15), 272, doi:10.21105/joss.00272

{\em Tissue resolved, gene structure refined equine transcriptome.} A Mansour, EY Scott, CJ Finno, RR Bellone, MJ Mienaltowski, MC Penedo, PJ Ross, SJ Valberg, JD Murray, CT Brown. BMC Genomics 18 (1) p103. 2017.

{\em A new chicken genome assembly provides insight into avian genome structure}. WC Warren et al. G3 7 (1) 109-117. 2017.

{\em Transcriptome of the Caribbean stony coral Porites astreoides from three developmental stages.} Mansour TC, Rosenthal JJC, Brown CT, Roberson LM.
GigaScience 5 (1), 33, August 2016.

{\em How open science helps researchers succeed.}
McKiernan EC et al., eLife 5, e16800, July 2016.

{\em Haplotype-phased synthetic long reads from short-read sequencing.}
JA Stapleton, et al.,
PLOS One 11 (1), e0147229, 2016.

{\em Microbial community analysis with ribosomal gene fragments from shotgun metagenomes.} Guo J, Cole JC, Zhang Q, Brown CT, Tiedje JM. Applied and environmental microbiology. Jan 2016.

{\em Haplotype-phased synthetic long reads from short-read sequencing.} Stapleton JA, Kim J, Hamilton JP, Wu M, Irber LC, Maddamsetti R, Briney B, Newton L, Burton DR, Brown CT, Chan C, Buell CR, Whitehead TA. PLOS One. Jan 2016.

{\em Hsp90 and hepatobiliary transformation during sea lamprey metamorphosis.}
Chung-Davidson YW, Yeh CY, Bussy U, Li K, Davidson PJ, Nanlohy KG, Brown CT, Whyard S, Li W. BMC Developmental Biology. Dec 2015.

{\em A new method for DNA sequencing error verification and correction via an on-disk index tree}. Gu Y, Liu X, Zhu Q, Dong Y, Brown CT, Pramanik S.  Proceedings of the 6th ACM Conference on Bioinformatics, Computational Biology and Health Informatics. Sep 2015.

{\em Differentially-Expressed Pseudogenes in HIV-1 Infection.} Gupta A, Brown CT, Zheng YH, Adami C. Viruses. Sep 2015.

\vspace{1cm}
{\centerline {Page 4/9}}

\newpage

{\em The khmer software package: enabling efficient nucleotide sequence analysis.} Crusoe MR et al., Brown CT. F1000Research. Sep 2015.

{\em Xander: employing a novel method for efficient gene-targeted metagenomic assembly.} Wang Q, Fish JA, Gilman M, Sun Y, Brown CT, Tiedje JM, Cole JR. Microbiome. August 2015.

{\em Transcriptome variation in response to Marekâ€™s disease virus acute infection.} Preeyanon L, Brown CT, Cheng H. Cytogenetics. July 2015.

{\em Phylogeny and phylogeography of functional genes shared among seven terrestrial subsurface metagenomes reveal N-cycling and microbial evolutionary relationships.} Lau MCY et al., Onstott TC. Frontiers in Microbiology. Oct 2014.

{\em Divergent mechanisms regulate conserved cardiopharyngeal development and gene expression in distantly related ascidians.} Stolfi A, Lowe EK, Racioppi C, Ristoratore F, Brown CT, Swalla BJ, Christiaen L. eLife. Sep 2014.

{\em These are not the k-mers you are looking for: efficient online
  k-mer counting using a probabilistic data structure.} Zhang Q, Pell
J, Canino-Koning R, Howe AC, {\bf Brown CT}.  PLoS One. July 2014.

{\color{red} ${\bf \triangleright}$}
{\em Tackling soil diversity with the assembly of large, complex metagenomes.}
Howe AC, Jansson J, Malfatti SA, Tringe SG, Tiedje JM, {\bf Brown CT}. Accepted at PNAS, 2/2014.  preprint arXiv:1212.2832.

{\em The Ribosomal Database Project: Data and Tools for High Throughput rRNA Analysis.} Cole JR, Wang Q, Fish J, Chai B, McGarrell D, Sun Y, Brown CT, Porras-Alfaro A, Kuske C, Tiedje JM.  Accepted, Nucleic Acid Res, Nov 2013.

{\em Genomic versatility and functional variation between two dominant
heterotrophic symbionts of deep-sea {\rm Osedax} worms.} Goffredi S, Yi H, Zhang Q, Klann J, Struve I, Vrijenhoek RC, {\bf Brown CT}. Accepted, ISME Journal, October 2013.

{\em Best practices for scientific computing.} Wilson GV et al. preprint arXiv:1210.0530. Accepted PLoS Biology, October 2013.

{\em FunGene: the Functional Gene Pipeline and Repository.} Fish JA, Chai B, Wang Q, Sun Y, {\bf Brown C}, Tiedje JM and Cole JR (2013).  Front. Microbiol. 4:291. doi: 10.3389/fmicb.2013.00291

{\em The genome and developmental transcriptome of the strongylid
  nematode Haemonchus contortus.}  Schwarz EM, Korhonen PK, Campbell
BE, Young ND, Jex AR, Jabbar A, Hall RS, Mondal A, Howe AC, Pell J,
Hofmann A, Boag PR, Zhu XQ, Gregory TR, Loukas A, Williams BA,
Antoshechkin I, {\bf Brown CT}, Sternberg PW, Gasser RB.  Genome
Biol. 2013 Aug 28;14(8):R89.

{\em The sea lamprey has a primordial accessory olfactory system.}
Chang S, Chung-Davidson YW, Libants SV, Nanlohy KG, Kiupel M, {\bf
  Brown C.T.}, Li W. BMC Evol Biol. 2013 Aug 17;13(1):172. doi:
10.1186/1471-2148-13-172.

{\em Integrated Analyses of Genome-Wide DNA Occupancy and Expression
  Profiling Identify Key Genes and Pathways Involved in Cellular
  Transformation by a Marek's Disease Virus Oncoprotein, Meq.}
Subramaniam S, Johnston J, Preeyanon L, {\bf Brown CT}, Kung HJ, Cheng
HH. J Virol. 2013 Aug;87(16):9016-29. doi: 10.1128/JVI.01163-13.

\vspace{1cm}
{\centerline {Page 5/9}}

\newpage

{\em A thermogenic secondary sexual character in male sea lamprey.}
Chung-Davidson, Y.-W., Priess, M.C., Yeh, C.-Y., Brant, C.O., Johnson,
N.S., Li, K., Nanlohy, K.G., Bryan, M.B., {\bf Brown, C.T.}, Choi, J.,
Li, W. Journal of Experimental Biology. 2013 Jul 15;216(Pt
14):2702-12. doi: 10.1242/jeb.085746.

{\em Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.} Smith JJ et al. Nature Genetics, Apr;45(4):415-21, 421e1-2. doi: 10.1038/ng.2568. Epub 2013 Feb 24.

{\em Draft Genome Sequences of two Campylobacter jejuni Clinical Isolates,
NW and D2600.} Jerome JP, Klahn BD, Bell JA, Barrick JE, {\bf Brown CT}, Mansfield LS. Journal of Bacteriology, 194 (20), 5707-5708.

{\em Standing Genetic Variation in Contingency Loci Drives the Rapid
  Adaptation of Campylobacter jejuni to a Novel Host} Jerome JP, Bell
JA, Plovanich-Jones AE, Barrick JE, {\bf Brown CT}, Mansfield LS.  PLoS One 6
(1), e16399, Jan 24 2011.

{\em Exploring the future of bioinformatics data sharing and mining
  with Pygr and Worldbase} Lee C, Alekseyenko A, {\bf Brown CT}.  in {\em
  Proceedings of the 8th Python in Science conference (SciPy 2009)}, G
Varoquaux, S van der Walt, J Millman (Eds.), pp. 62-67.

{\em Diverse syntrophic partnerships from deep-sea methane vents revealed by direct cell capture and metagenomics.}\\
Pernthaler A, Dekas AE, {\bf Brown CT}, Goffredi SK, Embaye T, Orphan VJ.\\
Proc Natl Acad Sci U S A. 2008 May
13;105(19):7052-7. Epub 2008 May 8.  PMID: 18467493.

{\em The genome of the sea urchin Strongylocentrotus purpuratus.}\\
Sea Urchin Genome Sequencing Consortium.\\
Science. 2006 Nov 10;314(5801):941-52.
PMID: 17095691.

{\em High regulatory gene use in sea urchin embryogenesis: Implications for bilaterian development and evolution.}\\
Howard-Ashby M, Materna SC, {\bf Brown CT}, Tu Q, Oliveri P, Cameron RA, Davidson EH.\\
Dev Biol. 2006 Dec 1;300(1):27-34. Epub 2006 Oct 18.
PMID: 17101125.

{\em Gene families encoding transcription factors expressed in early development of Strongylocentrotus purpuratus.}
Howard-Ashby M, Materna SC, {\bf Brown CT}, Chen L, Cameron RA, Davidson EH.\\
Dev Bio 2006 300 (1), 90-107.

{\em Identification and characterization of homeobox transcription factor genes in Strongylocentrotus purpuratus, and their expression in embryonic development}
Howard-Ashby M, Materna C, {\bf Brown CT}, Chen L, Cameron RA, Davidson EH.
2006 Dev Bio 300 (1), 74-89.

{\em Sea urchin Forkhead gene family: phylogeny and embryonic expression}
Tu Q, {\bf Brown CT}, Davidson EH, Oliveri P.
2006 Dev Bio 300 (1), 49-62.

{\em Paircomp, FamilyRelationsII and Cartwheel: tools for interspecific sequence comparison.}\\
{\bf Brown CT}, Xie Y, Davidson EH, Cameron RA.\\
BMC Bioinformatics. 2005 Mar 24;6:70.
PMID: 15790396.

{\em Anaerobic regulation by an
atypical Arc system in {\em Shewanella oneidensis}.}\\
Gralnick JA, {\bf Brown CT}, Newman DK.\\
Mol Microbiol. 2005
Jun;56(5):1347-57.  PMID: 15882425.

\vspace{2cm}
{\centerline {Page 6/9}}

\newpage

{\color{red} ${\bf \triangleright}$}
{\em Evolutionary comparisons suggest
many novel cAMP response protein binding sites in {\em E. coli}.}\\
{\bf Brown CT}, Callan CG Jr.\\
Proc Natl Acad Sci U S A. 2004 Feb 24;101(8):2404-9.  PMID: 14983022.

{\em Patchy interspecific sequence similarities efficiently identify positive cis-regulatory elements in the sea urchin.}
Yuh CH, {\bf Brown CT}, Livi CB, Rowen L, Clarke PJC, Davidson EH.
2002 Dev Bio 246 (1), 148-161.

  {\em New
computational approaches for analysis of {\em cis}-regulatory networks.}  \\
{\bf Brown CT}, Rust AG, Clarke PJ, Pan Z, Schilstra MJ, De Buysscher
T, Griffin G, Wold BJ, Cameron RA, Davidson EH, Bolouri H.\\
Dev Biol. 2002 Jun 1;246(1):86-102.  PMID: 12027436.

{\em A genomic
regulatory network for development.}\\
Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH,
Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, {\bf Brown CT}, Livi
CB, Lee PY, Revilla R, Rust AG, Pan Z, Schilstra MJ, Clarke PJ, Arnone
MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri H.
\\Science. 2002 Mar
1;295(5560):1669-78.  PMID: 11872831.

{\em A provisional regulatory gene network for specification of endomesoderm in the sea urchin embryo.}
Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH, Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, Brown CT, Livi CB, Lee PY, Revilla R, Schilstra MJ, Clarke PJ, Rust AG, Pan Z, Arnone MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri H.
Dev Biol. 2002 Jun 1;246(1):162-90.
PMID: 12027441.

{\em The Earthshine Project: update on photometric and spectroscopic measurements.}
 E. Palle, P. M. Rodriguez, P. R. Goode, J. Qiu, V. Yurchyshyn, J. Hickey, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
Solar Variability and Climate Change Advances in Space Research 34, 288 (2004).

{\em The earthshine spectrum}
P. M. Rodriguez, E. Palle, P. R. Goode, J. Hickey, J. Qiu, V. Yurchyshyn, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
Solar Variability and Climate Change Advances in Space Research 34, 293 (2004).

{\em Sunshine, Earthshine and Climate Change: II. Solar Origins of Variations in the Earth's Albedo.}  P.R. Goode, E. Palli,V. Yurchyshyn, J. Qiu,
 J. Hickey, P. Montaqis-Rodriguez,M.-C. Chu,
 E. Kolbe,C.T. Brown, S.E. Koonin.
Journal of the Korean Astronomical Society, 35, 1 (2003).

{\em Earthshine and the Earth's albedo: 1. Earthshine observations and measurements of the lunar phase function for accurate measurements of the Earth's Bond albedo}
J. Qiu, P. R. Goode, E. Palle, V. Yurchyshyn, J. Hickey, P. M. Rodriguez, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
J. of Geophys. Res.-Atmospheres 108, 4709 (2003).

{\em Earthshine and the Earth's albedo: 2. Observations and simulations over three years}
 E. Palle, P. R. Goode, V. Yurchyshyn, J. Qiu, J. Hickey, P. M. Rodriguez, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin.
J. of Geophys. Res.-Atmospheres 108, 4710 (2003).


{\em Earthshine observations of the earth's reflectance}
P. R. Goode, J. Qiu, V. Yurchyshyn, J. Hickey, M.-C. Chu, E. Kolbe, C. T. Brown, and S. E. Koonin
Geophys. Res. Lett. 28, 1671 (2001).

\vspace{1cm}
{\centerline {Page 7/9}}

\newpage

{\em Visualizing Evolutionary Activity of Genotypes in Evita};
with M. Bedau.  Adaptive Systems, 1998.

{\em A Comparison of Evolutionary Activity in Artificial Living Systems and in the 
	Biosphere;} Snyder E, Brown CT, Bedau M, Packard N.
in the Proceedings of the 4th Europ. Conf. on 
Artificial Life, July, 1997.

{\em Abundance Distributions in Artificial Life and Stochastic Models: "Age and
Area" revisited}, Adami, C., Brown, C.T., Haggerty, M.R.
Proc. of 3rd Europ. Conf. on Artificial Life, June 4-6, 1995,
Granada, Spain, Lecture Notes in Computer Science, Springer Verlag (1995),
p.503.

{\em Evolutionary Learning in the 2D Artificial Life System ``Avida''}\\
Adami C, Brown CT. Proc. of ``Artificial Life IV'', MIT Press, p. 377-381
(1994).

\section{\mysidestyle Invited Articles and Reviews}

{\em Strain recovery from metagenomes. Brown CT.} Nature Biotechnology
33 (10), 1041-1043, Oct 2015.

{\em Metagenomics: the paths forward}.  Brown CT and Tiedje JM.
Handbook of Molecular Microbiology II: Metagenomics in Different
Habitats.  Wiley-Blackwell 10 Nov 2011.

{\em Computational approaches to finding and analyzing cis-regulatory elements.} 
Brown CT. Methods Cell Biol. 2008;87:337-65. Review.
PMID: 18485306.

{\em Reproducible Bioinformatics Research for Biologists}.  Preeyanon
L, Pyrkosz AB, and Brown CT. Chapter in Implementing Reproducible
Computational Research, V. Stodden and R. Peng, ed.  Forthcoming in
Dec 2013.

\section{\mysidestyle Reports and Editorials}

{\em The Architecture of the LISA Science Analysis.} SA Teukolsky et al.
Keck Institute for Space Studies

{\em The principles of tomorrow's university.} DS Katz et al. F1000Research 7

{\em Keeping it light:(Re) analyzing community-wide datasets without major infrastructure.} H Alexander, LK Johnson, {\bf CT Brown}.
GigaScience.

{\em Cephalopod genomics: A plan of strategies and organization.} Albertin et al., Standards in Genomic Sciences 7 (1), 175.

{\em Changing computational research. The challenges ahead.}
Neylon, C., Aerts, J., Brown, C.T., Lemire, D., Millman, J., Murray-Rust, P., Perez, F., Saunders, N., Smith, A., Varoquaux, G.
Source Code for Biology and Medicine 7 (1), 2 (2012).

{\em Meeting report: the terabase metagenomics workshop and the vision of an Earth microbiome project.}
Gilbert, J.A., Meyer, F., Antonopoulos, D., Balaji, P., Brown, C.T., Brown, C.T., Desai, N., Eisen, J.A., Evers, D., Field, D. Standards in genomic sciences, 3(3) 243 (2010).

\section{\mysidestyle Online commentaries, blogs, and social media}

Personal/professional blog at: \href{http://ivory.idyll.org/blog/}{ivory.idyll.org/blog/}.  A few selected posts (click on links): \href{http://ivory.idyll.org/blog/replication-i.html}{``Our approach to replication in computational science''}, \href{http://ivory.idyll.org/blog/thoughts-on-assemblathon-2.html}{``Thoughts on Assemblathon 2''}, \href{http://ivory.idyll.org/blog/the-future-of-khmer-2013-version.html}{``The future of khmer (2013)''}.

Twitter: \href{http://twitter.com/ctitusbrown}{@ctitusbrown}

\href{http://blogs.biomedcentral.com/bmcblog/2013/02/28/version-control-for-scientific-research/}{BioMedCentral invited blog post: ``Version control for scientific research''}

\vspace{2cm}
{\centerline {Page 8/9}}
\newpage

    %____________________________________________________________________________________
    \section{\mysidestyle Professional\\Activities}

\begin{itemize}
\item iPlant Scientific Advisory Board member (2014-2018).
\item Software Carpentry Scientific Advisory Board member (through 2015).
\item Accredited Software Carpentry Instructor.
\item Cephalopod Genome Sequencing Consortium Steering Committee, 2012-present.
\item BEACON NSF STC, Thrust Group co-leader (responsible for reviewing
proposals, organizing activities), 2010-2014.
\item Course director, 2010-2015, Next-Generation Sequence Analysis for Biologists, KBS, MSU.
\item Director, Data Intensive Biology Summer Institute 2017-present, UC Davis.
\item Course faculty, 2006-2008, Embryology Course, Woods Hole Marine Biological Laboratory.
\item Course faculty, 2015-present, STAMPS Course, Woods Hole Marine Biological Laboratory.
\item Development and maintenance of several open source bioinformatics tools, including
Cartwheel server for comparative sequence analysis, khmer k-mer software,
screed, and sourmash; github.com/dib-lab/.
\item Reviewer for National Science Foundation, National Institutes of Health, Sloan Foundation, Moore Foundation, Chan-Zuckerberg Initiative; Developmental Biology, BMC Bioinformatics, BMC Genomics, Genome Biology, Bioinformatics, PLoS One, etc.
\item Volunteer community moderator for python-ideas mailing list.
\item SAB memberships: ELIXIR-UK (2015-2018), CZI Human Cell Atlas (2019-present, Alliance for Genome Resources (2020-present)
\item Co-lead, NIH Data Commons Pilot Phase Consortium, November 2017-February 2019.
\item Engagement and training lead, NIH Common Fund Data Ecosystem, April 2019-present.
\end{itemize}

\section{\mysidestyle Teaching and Workshops}

\begin{itemize}

\item Open Problems in Bioinformatics, CSE/MMG graduate seminar
  course (2008-2009)
\item Database-Backed Web Development, CSE 4xx (2008-, yearly), at MSU.
\item Introduction to Computational Science for Evolutionary Biologists, CSE 801 (was 891) (2010-2014), at MSU.
\item Analyzing Next-Generation Sequencing Data, research workshop (2010-, yearly).
\item Software Carpentry workshops: Scripps Research Institute (11/2012), U. Arizona (4/2013)
\item Instructor at Marine Biological Laboratory course on Strategies and Techniques for Analyzing Microbial Population Structures, 2012 and 2013.
\item Co-instructor for Workshop on Microbial Bioinformatics, 10/2013, Caltech.
\item Lead instructor for Workshop on mRNAseq for Biologists, and Workshop for Advanced Bioinformatics Developers, 11/2013, The Centre For Genome Analysis, Norwich, UK.
\item Intro to Genetics and Genomics lab, GGG 201(b), UC Davis, 2017-present.
\item[*] Tools to Support Data Intensive Research, GGG 298, UC Davis, 2020.
\end{itemize}

\section{\mysidestyle References}

{\em (Contact details available upon request.)}

Dr. Ewan Birney, Associate Director of the EMBL-EBI.

Professor Jonathan Eisen, University of California Davis.

Professor Paul W. Sternberg, California Institute of Technology.

Professor Billie J. Swalla, University of Washington at Seattle.

{\centerline {Page 9/9}}

\end{resume}

\end{document}

%________________________________________________________________________________________
% EOF


